### PR TITLE
Clean up and break up integ test

### PR DESCRIPTION
**Pull request by reinsch82**
Fixes 
 - #28244 

### Reviewing cheatsheet

Before merging the PR, comments starting with 
- ❌ ❓**must** be fixed
- 🤔 💅 **should** be fixed
- 💭 **may** be fixed
- 🎉 celebrate happy things

### CODE REVIEW

**Comment by reinsch82** [comment]
@bot-gradle test RFN

**Review by donat — CHANGES_REQUESTED** [review]

File: subprojects/core/src/integTest/groovy/org/gradle/execution/commandline/CommandLineIntegrationTest.groovy [hunk 1]
```diff
@@ -219,25 +278,29 @@ class CommandLineIntegrationTest extends AbstractIntegrationTest {
...
 
-    @Test
     @Requires(IntegTestPreconditions.NotEmbeddedExecutor)
-    void usesScriptBaseNameAsApplicationNameForUseInLogMessages() {
+    def "uses script basename as application name for use in log messages"() {
+        when:
         def binDir = distribution.gradleHomeDir.file('bin')
         def newScript = binDir.file(OperatingSystem.current().getScriptName('my app'))
+
+        then:
```
[the remark this answers is not in the document]
Comment (reinsch82) [reply subprojects/core/src/integTest/groovy/org/gradle/execution/commandline/CommandLineIntegrationTest.groovy:303]: good point, I wasn't aware of `cleanup`

**Review by donat — APPROVED** [review]

**Comment by reinsch82** [comment]
@bot-gradle merge

**Comment by donat** [comment]
@bot-gradle merge

**Comment by reinsch82** [comment]
@bot-gradle test this

**Comment by reinsch82** [comment]
@bot-gradle test RFN

**Comment by reinsch82** [comment]
@bot-gradle test RFN

**Comment by reinsch82** [comment]
@bot-gradle test RFN

**Comment by reinsch82** [comment]
@bot-gradle test this

**Comment by reinsch82** [comment]
@bot-gradle test this

**Comment by donat** [comment]
🎉 🎉 🎉

**Pull request merged**